modules/mobile/app/controllers/mobile/v0/community_care_providers_controller.rb

```diff
@@ -52,8 +51,8 @@ def coordinates
         Mobile::FacilitiesHelper.user_address_coordinates(@current_user)
       end
 
+      # this doesn't appear to be in active use, presumably because the FE doesn't send the facility id
```

Comment:
this is for appt scheduling which didn't make the cut atm for the app so makes sense it's not being called. it will get called if scheduling does get used though

Reply:
The logging above `CC providers call start` is being logged, so it does seem to be in use somehow. I assume you mean the front end doesn't send facility id because that was only needed for appt scheduling.

Reply:
could that logging be coming from the backend mobile tester?

Reply:
I think you're right. I assumed I was looking at prod but all of these are for staging.